source/crud/tests/v2/aggregate-out-readConcern.yml

```diff
@@ -26,15 +26,15 @@ tests:
         command_started_event:
           command:
             aggregate: *collection_name
-            pipeline:
+            pipeline: &pipeline
               - $sort: { x: 1 }
               - $match: { _id: { $gt: 1 } }
-              - $out: *output_collection
+              - $out: "other_test_collection"
             readConcern: { level: "majority" }
     outcome:
       collection:
         name: *output_collection
-        data:
+        data: &data
```

Comment:
Would it make sense to define an `&outcome` anchor instead of this? I just noticed that the `outcome` blocks below are all the same.